internal/testbed/integration/k8senrichment/e2e_test.go

```diff
@@ -56,7 +61,7 @@ func TestE2E_ClusterRBAC(t *testing.T) {
 		ManifestsDir: filepath.Join(testDir, "telemetrygen"),
 		TestID:       testID,
 		OtlpEndpoint: fmt.Sprintf("otelcol-%s.%s:4317", testID, testNs),
-		DataTypes:    []string{"metrics", "logs", "traces"},
```

Comment:
k8senrichment test has a pipeline only for traces, so creating telemetrygen objects for metrics and logs does not have any value